apps/explorer/src/ui/collapsible/CollapsibleSection.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { ChevronRight12 } from '@mysten/icons';
+import { Text } from '@mysten/ui';
+import * as Collapsible from '@radix-ui/react-collapsible';
+import clsx from 'clsx';
+import { type ReactNode, useMemo, useState } from 'react';
+
+import { Divider } from '~/ui/Divider';
+
+export interface CollapsibleSectionProps {
+	children: ReactNode;
+	defaultOpen?: boolean;
+	title?: string | ReactNode;
+	externalControl?: {
+		open: boolean;
+		setOpen: (open: boolean) => void;
```

Comment:
Probably make this `onOpenChange` so that it's normal controlled prop

Reply:
Updated to accept open/onOpenChange